### PR TITLE
unpinned cryptography version

### PR DESCRIPTION
**Pull request by rudradevpal**
Signed-off-by: Rudradev Pal <rudradev.pal@gmail.com>

**Description of the change:**
Unpinned cryptography python dependency for ansible-opeator #5739 

**Motivation for the change:**
As per the issue #5739 

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [ ] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [ ] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

File: images/ansible-operator-2.11-preview/Pipfile [hunk 1]
```diff
@@ -9,10 +9,7 @@ ansible-runner-http = "==1.0.0"
 ipaddress = "==1.0.23"
 openshift = "~=0.13.0"
 jmespath = "==0.10.0"
-# cryptography needs to be pinned to 3.3.2 as this is the last version
-# before its setup requires rust, which is not available via RPM in the
-# base image. This pin should be re-evaluated once the base image is updated.
-cryptography = "==3.3.2"
+cryptography = "*"
```
Comment (asmacdo) [inline images/ansible-operator-2.11-preview/Pipfile:12]: this can just be `cryptogtaphy`, no = necessary, will default to latest

**Review by asmacdo — CHANGES_REQUESTED** [review]

File: images/ansible-operator-2.11-preview/Pipfile (hunk 1 above)
Comment (rudradevpal) [reply images/ansible-operator-2.11-preview/Pipfile:12]: should i change it?

Comment (rudradevpal) [reply images/ansible-operator-2.11-preview/Pipfile:12]: If I change it to just `cryptography`, pipenv install fails with error: `pipenv.vendor.toml.decoder.TomlDecodeError: Key name found without value.`.

So I think better to go with current syntax as its key-pair. @asmacdo please suggest.

**Review by asmacdo — CHANGES_REQUESTED** [review]
lgtm after rebase

**Pull request closed without merging**